x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -773,4 +773,24 @@ public void testAggregateAliasInFilter() {
     public void testProjectUnresolvedAliasInFilter() {
         assertEquals("1:8: Unknown column [tni]", error("SELECT tni AS i FROM test WHERE i > 10 GROUP BY i"));
     }
+
+    public void testGeoShapeInWhereClause() {
```

Comment:
Can you add one more test where all three unsupported scenarios take place? `WHERE .... GROUP BY .... ORDER BY ....` so that all three messages are logged.

Reply:
I would have to two other shape fields to the mapping for that. The failure messages are grouped by their source node, which is shape in this case. So, only one failure message would show up. Do you think it is worth adding other shape fields?